### DISCUSSION TITLE
Api key showin on web site html code

### DISCUSSION THREAD

**Discussion by kyesil** [firebase/flutterfire#13180] [Q&A]
I put apikey  my html code. 
flutter build web output include my api key plain text. 
is this secure for share my website ?
![image](https://github.com/user-attachments/assets/f4dc14b2-040a-463f-9ffd-478d39272b23)

**Comment by Ehesp** [comment]
These are your client credentials which are expected to be public/viewable. You just need to add measures to prevent unauthorized access to your data, for example with security rules if using Firestore.

See here for more information https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public
